### PR TITLE
Feature/ayr 139

### PR DESCRIPTION
**Pull request by ailkhamov-zaizi**

### CODE REVIEW

**Review by olaapata — DISMISSED** [review]

File: .github/workflows/platform-deployment.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: Terraform platform deployment
+on:
+  push:
+    branches: [feature/AYR-139]
```
Comment (gmalanga) [inline .github/workflows/platform-deployment.yml:4]: I don't think this is correct, it should be main. Any push to the main has to trigger the workflow.

File: .github/workflows/platform-deployment.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    inputs:
+      environment:
+        type: choice
+        description: Choose the environment you want to deploy to from the dropdown list
+        options:
+          - dev
+
+jobs:
+  terraform-deployment:
+    uses: nationalarchives/da-ayr-gthub-actions/.github/workflows/terraform-enviromements-action.yml@feature/AYR-139
```
Comment (gmalanga) [inline .github/workflows/platform-deployment.yml:13]: Same as the above, I think this should be main and not feature

**Review by gmalanga — DISMISSED** [review]

File: terraform/environments/dev/providers.tf [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  ## OPTIONAL - in some environments we make use of CloudCustodian to tag all 
+  ##            resources with the id of its creator.  This section will
+  ##            ignore those tag changes so TF doesn't want to update things
+  ##            in a constant loop.
+  #ignore_tags {
+  #  keys = [ "Creator" ]
+  #}
+}
+
+provider "aws" {
```
Comment (gmalanga) [inline terraform/environments/dev/providers.tf]: Why do we require another provider? I think you should remove this one.

**Review by gmalanga — CHANGES_REQUESTED** [review]

File: terraform/environments/dev/main.tf [hunk 4]
```diff
@@ -0,0 +1,13 @@
+module "da-vpc" {
+  source = "../../modules/da-vpc"
+  vpc_cidr = var.vpc_cidr
+  environment = var.environment
+  enable_nat_gateway = var.enable_nat_gateway
+  project_name = var.project_name
+  aws_account_id = var.aws_account_id
+  providers = {
```
Comment (gmalanga) [inline terraform/environments/dev/main.tf]: This provider should be removed

**Review by gmalanga — DISMISSED** [review]

**Review by gmalanga — APPROVED** [review]

**Pull request closed without merging**